example/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: example
 description: An example app to show the use of Animation Text Kit.
 
 environment:
-  sdk: ">=2.10.0 <3.0.0"
+  sdk: '>=2.12.0-259.12.beta <3.0.0'
```

Comment:
I'd like to see double-quotes used because that matches the _Format on Save_ default for `yaml`.

I wonder if this should be so specific and instead just say:
```yaml
sdk: ">=2.12.0 <3.0.0"
```